<?xml version="1.0"?>
<launch>
    <!-- ros_args.launch -->
    <arg name="file_name" default="$(find graph_map)/maps/ndt_map.MAP" />
    <arg name="map_topic" default="/maps/map_velodyne" />
    <!--node pkg="graph_map" type="graph_map_server" name="graph_map_server" args="" output="screen">
  <param name="map_topic" value="$(arg map_topic)"/>
  <param name="file_name" value="$(arg file_name)"/>
</node-->
    <!--node pkg="rviz" type="rviz" name="rviz" args="-d $(find graph_map)/rviz/showMap.rviz" /-->
    <node pkg="tf" type="static_transform_publisher" name="link1_broadcaster" args="0 0 0 0 0 0 world map 100" />

</launch>